### PR TITLE
small improvements from the code review of #587

### PR DESCRIPTION
**Pull request by jeremyschlatter**
## Before Opening a PR
- [x] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [x] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

**Comment by jeremyschlatter** [comment]
cc @bryanculbertson

**Review by bryanculbertson — APPROVED** [review]
Approving, but will wait to merge until you add an `r` to the regex

**Comment by bryanculbertson** [comment]
@jeremyschlatter Thank you for the followup PR!

File: vaccine_feed_ingest/runners/mn/gov/normalize.py [hunk 1]
```diff
@@ -58,8 +59,7 @@ def cleanup_url(url):
             return None
         if url == "COVID-19 Vaccine Information - Hennepin Healthcare":
             return None
-        if url == "https/:www.Lewisdrug.com":
-            url = "https://www.lewisdrug.com"
+        url = re.sub("^https/:", "https://", url)  # fix typos
```
[the remark this answers is not in the document]
Comment (jeremyschlatter) [reply vaccine_feed_ingest/runners/mn/gov/normalize.py]: Done.

**Pull request merged**